runtime_config_linux.go

```diff
@@ -21,6 +21,9 @@ type LinuxRuntime struct {
 	// Resources contain cgroup information for handling resource constraints
 	// for the container
 	Resources Resources `json:"resources"`
+	// CgroupsPath specifies the path to cgroups that are created and/or joined by the container.
+	// The path is expected to be relative to the cgroups mountpoint.
+	CgroupsPath string `json:"cgroupsPath"`
```

Comment:
@mrunalp: Your reasoning is sound! I see use cases for both `exec` and `resource updates`. Would making `Resources` optional resolve the issue?